contracts/test/compensation/compensationClaims.js

```diff
@@ -44,7 +37,7 @@ describe("Compensation Claims", async () => {
     const { governor, adjuster, compensationClaims } = fixture;
     let accounts = [];
     let amounts = [];
-    for (row of claims) {
+    for (let row of claims) {
```

Comment:
nit: how about const since row does not get re-assigned?

Reply:
👍